syn/syn_yosys.sh

```diff
@@ -4,50 +4,56 @@
 # Licensed under the Apache License, Version 2.0, see LICENSE for details.
 # SPDX-License-Identifier: Apache-2.0
 
-# This script converts all SystemVerilog RTL files to Verilog and then
-# runs Yosys for ibex_core
-#
-# The following tools are required:
-#  - sv2v: SystemVerilog-to-Verilog converter from github.com/zachjs/sv2v
-#  - yosys: synthesis tool from github.com/YosysHQ/yosys
-#
-# Usage:
-#   syn_yosys.sh 2>&1 | tee syn.std
-#
-# Above command generates two files:
-#   ibex_core_premap.v  : premap netlist (before mapping it onto std. cells)
-#   ibex_core_netlist.v : final netlist
-
-# TODO: below should be replaced by elegant fusesoc commands
+# This script drives the experimental Ibex synthesis flow. More details can be
+# found in README.md
```

Comment:
Did I miss this README file?

Reply:
Somehow got lost in my last minute fixups and force pushses (including lost off my local disk, luckily the reflog exists!)